app/assets/stylesheets/vendor/main.css.erb

```diff
@@ -711,7 +711,7 @@ aside .module span { color: #000; }
     transition: 0.3s ease;
 }
 
-.btn:hover, .btn a:hover, .shareSave .btn a:hover, .menu-btn:hover, .toggle:hover {
+.btn:hover, .btn a:hover, .shareSave .btn a:hover, .toggle:hover {
```

Comment:
I got rid of it because, on a mobile device, the "hover" state stays activated after you click on the link. It turns black and doesn't turn orange again. There may be some better way of handling it, but I was trying not to spend too much time on it.